src/abstractmps.jl

```diff
@@ -1687,10 +1692,13 @@ function truncate!(
   orthogonalize!(M, last(site_range))
 
   # Perform truncations in a right-to-left sweep
-  for j in reverse((first(site_range) + 1):last(site_range))
+  for (i,j) in enumerate(reverse((first(site_range) + 1):last(site_range)))
```

Comment:
```suggestion
  js = reverse((first(site_range) + 1):last(site_range))
  for i in eachindex(js)
    j = js[i]
```
Just as a style thing I'm not a big fan of `enumerate`.